File: app-java/src/main/java/com/danke/http/sample/UserResponse.java
package com.danke.http.sample;

import android.support.annotation.Keep;

import com.danke.http.response.BaseResponse;
import com.google.gson.annotations.SerializedName;

/**
 * @author danke
 * @date 2018/6/27
 */
@Keep
public class UserResponse extends BaseResponse<String> {

    // @SerializedName("type")
    // int type;

    @SerializedName("name")
    String name;

    @SerializedName("id")
    long id;

    @SerializedName("avatar_url")
    String avatar;

    @SerializedName("html_url")
    String url;
}

File: app-kotlin/src/main/kotlin/com/danke/http/sample/UserResponse.kt
package com.danke.http.sample

import android.support.annotation.Keep
import com.danke.http.response.BaseResponse
import com.google.gson.annotations.SerializedName

/**
 * @author danke
 * @date 2018/7/5
 */
@Keep
data class UserResponse(
        // @SerializedName("type") val type: Int,
        @SerializedName("name") val name: String?,
        @SerializedName("id") val id: Long,
        @SerializedName("avatar_url") val avatar: String?,
        @SerializedName("html_url") val url: String) : BaseResponse<String>()